modules/openshift-developer-cli-modifying-your-application-code.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * cli-reference/creating-an-application-with-odo.adoc     
+
+[id="modifying-your-application-code_{context}"]
+= Modifying your application code
+
+Edit your code and push the changes to the component. 
```

Comment:
This kind of just repeats the content of the steps below. Maybe say something instead like "You can modify your application code and have the changes applied to your application on {product-title}." or whatever exactly happens.